Doc/extending/windows.rst

```diff
@@ -125,6 +125,10 @@ The second command created :file:`ni.dll` (and :file:`.obj` and :file:`.lib`),
 which knows how to find the necessary functions from spam, and also from the
 Python executable.
 
+.. note::
+
+   The above commands are only applicable for Python 32-bit versions.
```

Comment:
It's not clear to me that this is actually true.  Either way, this entire section is woefully out of date (Borland, mentioned above, seems to have faded away nearly 20 years ago), and should be properly brought up to date rather than just sticking in a note block (which we generally try to use sparingly anyway).